net.nmlgc.rec98.sh01.yaml

```diff
@@ -0,0 +1,112 @@
+id: net.nmlgc.rec98.sh01
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm19
+build-options:
+  append-path: /usr/lib/sdk/llvm19/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm19/lib
+finish-args:
+  - --device=dri
+  - --device=input
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+cleanup:
+  - /bin/pango*
+  - /bin/tup*
+  - /include
+  - /lib/*.a
+  - /lib/libc++.modules.json
+  - /lib/pkgconfig
+  - /share/libc++
+  - /share/man
+
+command: GIAN07
+modules:
+  # See https://aur.archlinux.org/packages/ttf-ipa-mona
+  - name: ttf-ipa-mona
+    sources:
+      - type: archive
+        url: https://web.archive.org/web/20111108024313if_/http://www.geocities.jp/ipa_mona/opfc-ModuleHP-1.1.1_withIPAMonaFonts-1.0.8.tar.gz
+        dest-filename: ttf-ipa-mona.tar # It's actually uncompressed!
+        sha256: cb7b6975faf5caa070ffa1699a241d1c093f3b74c54696e5af23b5bc3b118a44
+    buildsystem: simple
+    subdir: fonts
+    build-commands:
+      - install -Dm644 ipag-mona.ttf -t "$FLATPAK_DEST/share/fonts/TTF/"
+      - install -Dm644 COPYING.font.ja -t "$FLATPAK_DEST/share/licenses/ttf-ipa-mona/"
+
+  - name: pango
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/pango/-/archive/1.56.0/pango-1.56.0.tar.bz2
```

Comment:
pango 1.54 isn't enough? It's in the platform.

Reply:
This is all about the patch to work around https://github.com/harfbuzz/harfbuzz/issues/4311. I figured I'd go for the latest version if I have to compile from source anyway. But I could try compiling Pango 1.54 instead if it's better to stick closer to the version in the platform.